proto/src/constraints.proto

```diff
@@ -32,10 +36,11 @@ message SummaryConstraintMsg {
   oneof second {
      string second_field = 3;
      double value = 4;
-     SummaryBetweenConstraintMsg between = 5;
```

Comment:
Can you explain the code changes here?

Reply:
Of course: the ones with the op, verbose and between are clear by now, as the answer to the question at the PR concerning the same, should be sufficient.
google.protobug.ListValue reference_set = 8; --> this one is used for the set provided by the user to be used with the IN, CONTAIN, and EQ set operations. As it can contain more than one data type, this was the possible solution.
We coordinated with @MilenaTrajanoska to use the same field too: https://github.com/whylabs/whylogs/pull/375/files/49c9e48adbe91126aba107bd595a4c47b0d1a315#diff-3ead279fdd02046f469418a748586aa737b3a3cf99be981b99db1bee165a2a22